secboot/secboot_tpm.go

```diff
@@ -338,14 +340,14 @@ func SealKeys(keys []SealKeyRequest, params *SealKeysParams) error {
 
 	authKey, err := sbSealKeyToTPMMultiple(tpm, sbKeys, &creationParams)
 	if err != nil {
+		logger.Debugf("seal key error: %v", err)
```

Comment:
Noticef maybe? Unless the returned err is shown anyway in the log?

Reply:
Yes, the error bubbles up and is shown in the log.

Reply:
Do we need Debugf then?